src/providers/sh/commands/deploy.js

```diff
@@ -127,6 +127,7 @@ const help = () => {
     -N, --forward-npm              Forward login information to install private npm modules
     --session-affinity             Session affinity, \`ip\` or \`random\` (default) to control session affinity
     -T, --team                     Set a custom team scope
+    --regions                      Set regions
```

Comment:
Let's change `Set regions` to something better 👌

Reply:
"Set the default regions this deployment is enabled on"